pipeline/runtime/run.go

```diff
@@ -66,6 +67,9 @@ func executeRunStep(ctx context.Context, f RunFunc, r *api.StartStepRequest, out
 		}
 	}
 
+	outputSecretsFile := fmt.Sprintf("%s/%s-output-secrets.env", pipeline.SharedVolPath, step.ID)
```

Comment:
Can you add the same condition as L52 and add a `SecretVarFile` variable. If it's set, we will use that directly otherwise generate it similarly to what you have. This will help integrate with runner in the future.

Reply:
```
if r.SecretVarFile != "" {
     // set the env variable to r.SecretVarFile
}
```